### ISSUE TITLE
Printing to specific folder not possible

### ISSUE THREAD

**Issue by JulianMaartens** [google/cloud-print-connector#403]
If this is not the right place for posting this, sorry. I'm kinda new to Github.

When using the Cloud Print Connector (running as service) we can no longer print to a specific folder/box on our print machine. Example: we have group 1 and group 2. In the properties of the printer we set-up the settings so that prints will be sent to the folder 'group 1'  or 'group 2' on the print machine.

Unfortunately this does not work. Prints are sent to the 'main' directory on the print machine.
We used the Cloud Print Service before, but we stopped using it because the printers would go offline when logging out on the server.

In short: the Connector is working fine but I want the prints to be sent to the regarding group directories on the print machine.
